test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -46,6 +46,32 @@ TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_02_n) {
   EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
 }
 
+/**
+ * @brief Neural Network Model Construct wit Configuration File Test
+ * @return 0 success, -EINVAL if failed.
+ */
+TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_04_n) {
+  ml_nnmodel_h handle;
+  const char *model_conf = "/test/test.ini";
```

Comment:
I think it is good to clarify this test is negative test by changing `/test/test.ini` -> `/test/wrong_conf.ini`.

Reply:
That's good. I will apply.